src/App.tsx

```diff
@@ -1,24 +1,242 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header';
+import { Todo } from './types/Todo';
+import {
+  deleteTodo,
+  getTodos,
+  patchTodo,
+  postTodo,
+} from './api/todos';
+import { TodoList } from './components/TodoList';
+import { TodoFilter } from './components/TodoFilter';
+import { Notification } from './components/Notification';
+import { StatusTodos } from './types/StatusTodo';
 
-const USER_ID = 0;
+const USER_ID = 9946;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState(StatusTodos.ALL);
+  const [isError, setIsError] = useState(false);
+  const [query, setQuery] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+  const [isTodoId, setIsTodoId] = useState<number[] | null>([]);
```

Comment:
Do you really need an Array type here?
As I see, this state is about just one active Todo, is't it?
If I'am right it may be something like `activeTodoId: number`